3-networks/README.md

```diff
@@ -9,10 +9,39 @@ The purpose of this step is to:
 
 1. 0-bootstrap executed successfully.
 1. 1-org executed successfully.
-1. 2-environments/envs/[dev|nonprod|prod] executed successfully.
+1. 2-environments executed successfully.
+1. Obtain the value for the access_context_manager_policy_id variable. Can be obtained by running `gcloud access-context-manager policies list --organization YOUR-ORGANIZATION_ID`. It is the value under `name`.
 
 ## Usage
 
-Follow the instructions in the README.md files of each individual 3-networks/envs/ folder.
+### Setup to run via Cloud Build
+
+1. Clone repo `gcloud source repos clone gcp-networks --project=YOUR_CLOUD_BUILD_PROJECT_ID`
```

Comment:
nit: `YOUR_CLOUD_BUILD_PROJECT_ID` we should call out that this comes from stage `0-bootstrap` if we havent already.

Reply:
step `1-org` has the instruction for getting the correct value:  `(this is from terraform output from the previous section, 0-bootstrap)`.

Reply:
possible future feature request:  Would it work well if we labeled the cloud build project during creation so this could be programmatically queried?  Something like:
`gcloud source repos clone gcp-org \
--project=$(gcloud projects list --filter="(labels.application_name:cloudbuild \
AND labels.env:seed)" --format="value(project_id)")`

Reply:
@mikelaramie we could have the this problem too: https://github.com/terraform-google-modules/terraform-example-foundation/issues/113 . Maybe we cloud just help a little more and give the terraform command `terraform output cloudbuild_project_id` in step `1-org`

Reply:
I think #113 is just a byproduct of testing. irl, we will only have one seed right? But either approach sgtm.